flink-connector-oceanbase-cdc/src/main/java/com/ververica/cdc/connectors/oceanbase/table/OceanBaseTableSourceFactory.java

```diff
@@ -64,13 +64,22 @@ public class OceanBaseTableSourceFactory implements DynamicTableSourceFactory {
             ConfigOptions.key("database-name")
                     .stringType()
                     .noDefaultValue()
-                    .withDescription("Database name of OceanBase to monitor.");
+                    .withDescription(
+                            "Database name of OceanBase to monitor, should be regular expression. Only can be used with 'initial' mode.");
 
     public static final ConfigOption<String> TABLE_NAME =
             ConfigOptions.key("table-name")
                     .stringType()
                     .noDefaultValue()
-                    .withDescription("Table name of OceanBase to monitor.");
+                    .withDescription(
+                            "Table name of OceanBase to monitor, should be regular expression. Only can be used with 'initial' mode.");
+
+    public static final ConfigOption<String> TABLE_LIST =
+            ConfigOptions.key("table-list")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "List of full names of tables, separated by commas, e.g. \"db1.table1, db2.table2\".");
```

Comment:
For some compatibility reasons, I think we can just keep `TABLE_NAME` config option to accept "db1.table1, db2.table2".

Reply:
`database-name` and `table-name` are used as regex, and `table-list` is used as exact value, and they can also be used together. That's my original plan.

Reply:
Keep `database-name` and `table-name` in sql option will also make the configuration the same with stream api, which may be better than using different config in different mode.

Reply:
> Keep `database-name` and `table-name` in sql option will also make the configuration the same with stream api, which may be better than using different config in different mode.

Yes, I agree with that. 
However, for users, there will be some differences using different connectors, which may increase the learning cost.

Reply:
> > Keep `database-name` and `table-name` in sql option will also make the configuration the same with stream api, which may be better than using different config in different mode.
> 
> Yes, I agree with that. However, for users, there will be some differences using different connectors, which may increase the learning cost.

I understand your concern, I will add instruction in document to help users figure out how to set monitor list.